src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/wikipage/WikiArticleHomePage.java

```diff
@@ -6,7 +6,7 @@
 
 public class WikiArticleHomePage extends WikiBasePageObject {
 
-  private By wikiHomePageSpecificElement = By.cssSelector("#HOME_TOP_LEADERBOARD");
+  private By wikiHomePageSpecificElement = By.cssSelector(".mainpage");
```

Comment:
Thanks for sharing this! `wikiHomePageSpecificElement` is used in`wait.forElementPresent()` which in the end calls `findElement(locator, driver)`.  `findElement` method returns a WebElement object otherwise it throws an exception - so it always returns a first found element - simply checks for presence so any cssSelector is safe here. I've however changed `cssSelector` to `className` to be more fancy ;) all tests passes on jenkins.